assets/dataImportCronTemplates/dataImportCronTemplates.yaml

```diff
@@ -30,13 +34,14 @@
           resources:
             requests:
               storage: 10Gi
-    garbageCollect: Outdated
-    managedDataSource: centos-stream10
 - metadata:
     annotations:
       cdi.kubevirt.io/storage.bind.immediate.requested: "true"
+      ssp.kubevirt.io/dict.architectures: arm64,s390x,amd64
```

Comment:
why different order than the first two?

Reply:
1. the order is meaningless
2. this is done by the tool, introduced in #3523 - this tool takes the architectures from the manifest image, by the order of the manifest, meaning, this is the order in the manifest.